### DISCUSSION TITLE
How to pass field parameters to validators or sub validators

### DISCUSSION THREAD

**Discussion by breezechime** [pydantic/pydantic#9733] [Question]
For example, I want to set the version of the uuid validator

### example
```python
import uuid
from pydantic import BaseModel, ConfigDict, Field

class Test(BaseModel):

    model_config = ConfigDict(str_to_upper=True)

    uid: uuid.UUID = Field(version=1)

print(Test.__pydantic_validator__)
Test(uid='41a79522-316a-11ef-bf27-56f55720ded6')
```

### out validators schema
```bash
UuidValidator {
    strict: false,
    version: None,
},
```

But I got an output with version equal to empty, and the version I passed was not recognized
Another scenario is the sub validator
还有一种情况是子验证器

example: list[list[str]]
How can I set parameters such as max_length for the subitem list validator
我该如何为子项list验证器设置max_length之类的参数

**Comment by Viicos** [comment]
You are probably looking for https://docs.pydantic.dev/latest/api/types/#pydantic.types.UUID1.

**Comment by breezechime** [comment]
But this can only combine the parameters of the current validator. How should we pass the parameters of the sub validators? There are also cases where the sub validators are listed
但是这个只能组合当前验证器的参数，该如何传递子验证器的参数呢，也有些子验证器为列表情况的时候
